services/sidecar/tests/unit/test_log_parser.py

```diff
@@ -0,0 +1,65 @@
+from simcore_service_sidecar.log_parser import (
+    LogType,
+    parse_line,
+    monitor_logs_task,
+)
+import pytest
+from asyncio import Future, ensure_future, sleep
+from pathlib import Path
+
+
+@pytest.mark.parametrize(
+    "log, expected_log_type, expected_parsed_message",
+    [
```

Comment:
Nice parametrization. Helps extending to the many cases the users will be bringing